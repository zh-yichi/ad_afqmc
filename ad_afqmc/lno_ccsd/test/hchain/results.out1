# frag  mp2_orb_corr  ccsd_orb_corr  afqmc_hf_orb_en  err  afqmc_cc_orb_en  err  norb  nelec  time
  1  -0.01598222  -0.02530791  -0.024040  0.000626  -0.024040  0.000626  3  2  58.76
  2  -0.01705853  -0.02843001  -0.027399  0.000728  -0.027401  0.000729  2  2  51.21
  3  -0.01705853  -0.02843001  -0.027717  0.001317  -0.027719  0.001320  2  2  51.08
# final results 
# mean-field energy: -3.15900175
# lno-thresh [0.01, 0.001]
# e_mp2_corr: -0.050099
# e_ccsd_corr: -0.082168
# afqmc/hf_corr: -0.079156 +/- 0.001630
# afqmc/cc_corr: -0.079160 +/- 0.001633
# mp2_correction: -0.01238628
# number of orbitals: average 2.33 maxium 3
# number of electrons: average 2.00 maxium 2
# total run time: 161.05
