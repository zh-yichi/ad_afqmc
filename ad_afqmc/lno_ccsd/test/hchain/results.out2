# frag  mp2_orb_corr  ccsd_orb_corr  afqmc_hf_orb_en  err  afqmc_cc_orb_en  err  norb  nelec  time
  1  -0.01907257  None  -0.028104  0.000568  -0.028639  0.000062  5  4  40.94
  2  -0.02023033  None  -0.013637  0.020147  -0.033268  0.000018  4  4  38.01
  3  -0.02023033  None  -0.036512  0.001332  -0.033220  0.000020  4  4  38.24
# final results 
# mean-field energy: -3.15900175
# lno-thresh [0.001, 0.0001]
# e_mp2_corr: -0.059533
# e_ccsd_corr: 0.000000
# afqmc/hf_corr: -0.078253 +/- 0.020199
# afqmc/cc_corr: -0.095127 +/- 0.000068
# mp2_correction: -0.00295234
# number of orbitals: average 4.33 maxium 5
# number of electrons: average 4.00 maxium 4
# total run time: 117.19
