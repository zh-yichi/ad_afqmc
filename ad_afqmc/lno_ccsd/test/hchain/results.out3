# frag  mp2_orb_corr  ccsd_orb_corr  afqmc_hf_orb_en  err  afqmc_cc_orb_en  err  norb  nelec  time
  1  -0.02115507  None  -0.037077  0.002665  -0.034269  0.000045  6  6  52.85
  2  -0.02066525  None  -0.030507  0.010804  -0.034015  0.000049  6  6  52.98
  3  -0.02066525  None  -0.065037  0.022997  -0.034076  0.000066  6  6  52.98
# final results 
# mean-field energy: -3.15900175
# lno-thresh [9.999999999999999e-06, 1e-06]
# e_mp2_corr: -0.062486
# e_ccsd_corr: 0.000000
# afqmc/hf_corr: -0.132621 +/- 0.025548
# afqmc/cc_corr: -0.102360 +/- 0.000094
# mp2_correction: 0.00000001
# number of orbitals: average 6.00 maxium 6
# number of electrons: average 6.00 maxium 6
# total run time: 158.81
