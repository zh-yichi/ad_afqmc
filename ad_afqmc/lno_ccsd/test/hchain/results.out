# frag 	 elec_orb_cr 	 err 	 hf_orb_cr 	 err 	 ccsd_orb_cr 	 err 	 e_afqmc_orb_cr 	 err 	 e_mp2_orb_corr 	 e_ccsd_orb_corr
  1 	 0.219039 	 0.002411 	 -0.041821 	 0.000302 	 -0.222098 	 0.002150 	 -0.044879 	 0.000030 	 -0.035023 	 -0.044456
  2 	 0.110899 	 0.001221 	 -0.039457 	 0.000449 	 -0.112695 	 0.000931 	 -0.041253 	 0.000022 	 -0.031554 	 -0.041040
  3 	 0.110899 	 0.001221 	 -0.039457 	 0.000449 	 -0.112695 	 0.000931 	 -0.041253 	 0.000022 	 -0.031554 	 -0.041040
